src/app/examples/si-tabs/si-tabs-next-routing.html

```diff
@@ -0,0 +1,7 @@
+<si-tabset-next>
+  <a si-tab-next heading="Home" aria-label="Home" [routerLink]="tabLink1"></a>
```

Comment:
I guess you can drop the aria-label here. I guess it is just the linter complaining. Maybe adjust the aria rule in config. We already did this for something else I guess.

I will check if we may can bring those exceptions to simpl/lint